File: simba-create/src/main/java/com/simba/model/ProjectPackage.java
package com.simba.model;

public class ProjectPackage {

	/**
	 * 项目名称
	 */
	private String versionNo;

	/**
	 * 项目名称
	 */
	private String projectCode;

	/**
	 * 包名
	 */
	private String packageName;

	/**
	 * 超级管理员账号
	 */
	private String account;

	/**
	 * 超级管理员密码
	 */
	private String pwd;

	/**
	 * 普通用户密码
	 */
	private String defaultPwd;

	/**
	 * 加密秘钥
	 */
	private String encryptKey;

	/**
	 * 管理系统端口号
	 */
	private int adminPort;

	/**
	 * 用户系统端口号
	 */
	private int userPort;

	public int getAdminPort() {
		return adminPort;
	}

	public void setAdminPort(int adminPort) {
		this.adminPort = adminPort;
	}

	public int getUserPort() {
		return userPort;
	}

	public void setUserPort(int userPort) {
		this.userPort = userPort;
	}

	public String getVersionNo() {
		return versionNo;
	}

	public void setVersionNo(String versionNo) {
		this.versionNo = versionNo;
	}

	public String getProjectCode() {
		return projectCode;
	}

	public void setProjectCode(String projectCode) {
		this.projectCode = projectCode;
	}

	public String getPackageName() {
		return packageName;
	}

	public void setPackageName(String packageName) {
		this.packageName = packageName;
	}

	public String getAccount() {
		return account;
	}

	public void setAccount(String account) {
		this.account = account;
	}

	public String getPwd() {
		return pwd;
	}

	public void setPwd(String pwd) {
		this.pwd = pwd;
	}

	public String getDefaultPwd() {
		return defaultPwd;
	}

	public void setDefaultPwd(String defaultPwd) {
		this.defaultPwd = defaultPwd;
	}

	public String getEncryptKey() {
		return encryptKey;
	}

	public void setEncryptKey(String encryptKey) {
		this.encryptKey = encryptKey;
	}

}

File: simba-create/src/main/resources/static/js/projectPackage/projectPackage.js
var ProjectPackage = {

	"checkForm": function() {
		var versionNo = $("#versionNo").val();
		if(!versionNo) {
			parent.showInfo("版本号不能为空");
			return false;
		}
		var projectCode = $("#projectCode").val();
		if(!projectCode) {
			parent.showInfo("项目名称不能为空");
			return false;
		}
		var packageName = $("#packageName").val();
		if(!packageName) {
			parent.showInfo("包名不能为空");
			return false;
		}
		var account = $("#account").val();
		if(!account) {
			parent.showInfo("超级管理员账号不能为空");
			return false;
		}
		var pwd = $("#pwd").val();
		if(!pwd) {
			parent.showInfo("超级管理员密码不能为空");
			return false;
		}
		var defaultPwd = $("#defaultPwd").val();
		if(!defaultPwd) {
			parent.showInfo("普通用户密码不能为空");
			return false;
		}
		var encryptKey = $("#encryptKey").val();
		if(!encryptKey) {
			parent.showInfo("加密秘钥不能为空");
			return false;
		}
		var adminPort = $("#adminPort").val();
		if(!adminPort) {
			parent.showInfo("管理系统端口号不能为空");
			return false;
		}
		var userPort = $("#userPort").val();
		if(!userPort) {
			parent.showInfo("用户系统端口号不能为空");
			return false;
		}
		parent.showSuccessInfo("正在打包中，请耐心等待......");
		return true;
	},

	"end": null

};